src/java.base/share/classes/sun/util/resources/LocaleNames.properties

```diff
@@ -1094,7 +1094,7 @@ TL=Timor-Leste
 TM=Turkmenistan
 TN=Tunisia
 TO=Tonga
-TR=Turkey
+TR=T�rkiye
```

Comment:
I believe this is fine, but please confirm that this ends up correct u-umulaut when you get the text. It is somewhat a rare case where ISO-8859-1 has the character and the byte sequence differs between ISO-8859-1 and UTF-8.

Reply:
Thanks for pointing that out, I switched it to just be a unicode escaped ü, as that would be more consistent with the rest of the file. I also added the changes to the test files like you suggested.